### PR TITLE
Feature/update readme

### LINKED ISSUE #173: Update install instructions
According to the conversation in #152, some modifications should be performed to the install instructions. This is the summary up to now:

 - Move the `catkin build` part for the end, since it will now also build the Android part (so its pre-requisites need to go first)
 - Include `ros-kinetic-rosjava-build-tools` (or `ros-indigo-rosjava-build-tools`) as an additional dependent package to install
 - Include the instructions to add `local.properties` (and fix the trailing white space!) as it is done [here](https://github.com/Intermodalics/tango_ros/blob/master/TangoRosStreamer/README.md) 
- Use `catkin build --no-jobserver` to build the application. Specify the path for the resulting apk file.

### PR DESCRIPTION
**Pull request by PerrineAguiar**
This PR adds instruction to build the app using catkin. It also removes the readme of the TangoRosStreamer directory (all documentation is now in the main readme of the repo).

Fixes #173.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +38,7 @@ sudo rosdep init
 rosdep update
 echo "source /opt/ros/indigo/setup.bash" >> ~/.bashrc
 source ~/.bashrc
+sudo apt-get install ros-indigo-rosjava-build-tools
```
Comment (mcopejans) [inline README.md]: The app is compatible with both indigo and kinetic. Would be nice to abstract this as <ros-distro> or just make a note about this.

Comment (PerrineAguiar) [inline README.md]: There is already a note about this at the line below.

Comment (mcopejans) [inline README.md]: Oops missed that one sorry.

File: README.md [hunk 2]
```diff
@@ -8,9 +8,22 @@ This work is developed by [Intermodalics](http://www.intermodalics.eu/) in colla
 Do not hesitate to give us feedback if something is broken or if you think it lacks some features. The best way to do this is by adding issues to this repository.
 
 # Kickstart
-The app is available in Google's Play Store: https://play.google.com/store/apps/details?id=eu.intermodalics.tango_ros_streamer
+The app is available in Google's Play Store: https://play.google.com/store/apps/details?id=eu.intermodalics.tango_ros_streamer  
+It can be installed on any Tango-enable device. Note that the minimum Tango version required to run Tango Ros Streamer is Yildun (you will find the Tango release history [here](https://developers.google.com/tango/release-notes)). To check the Tango version of your device go to Settings->Apps->Tango Core.
```
Comment (jubeira) [inline README.md]: nit: "... any Tango-_enabled_ device"

**Pull request merged**